src/EditTableDialog.ui

```diff
@@ -99,7 +99,7 @@
      </property>
      <widget class="QTabWidget" name="groupDefinition">
       <property name="currentIndex">
-       <number>0</number>
+       <number>1</number>
```

Comment:
This should be left at 0 so the default tab is the Fields tab and not the Constraints tab.